config/base.yml

```diff
@@ -1065,19 +1168,23 @@ Style/SelfAssignment:
 
 Style/Semicolon:
   Enabled: true
+  AllowAsExpressionSeparator: false
 
 Style/SingleLineMethods:
   Enabled: true
+  AllowIfMethodIsEmpty: false
```

Comment:
@onebree this one is default to `true`, but to keep it consistent with standard's `Style/EmptyMethod`(expanded) I changed it to `false`